cmd/root.go

```diff
@@ -178,6 +178,13 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&rest.Domain, "domain", defaultDomain, "Your assigned Akita domain (e.g. company.akita.software)")
 	rootCmd.PersistentFlags().MarkHidden("domain")
 
+	// Use a proxy or permit an unknown certificate.
+	rootCmd.PersistentFlags().StringVar(&rest.UseProxy, "proxy", "", "The domain name or IP address of an HTTP proxy server to use")
```

Comment:
It appears that a user can specify `https://proxy.example.net` or `socks5://proxy.example.net`, and the implementation will connect through the HTTPS or SOCKS5 proxy. Shall we document and support these too?

Reply:
Yes, my intention was to support any URI (though we don't validate that it is only one of HTTP, HTTPS, or SOCKS.)

I will change the comment to include the URI as an option, but I think of the service being named as an "HTTP proxy" independent of the protocol.